src/test/meson.build

```diff
@@ -620,7 +620,7 @@ tests += [
          [libshared],
          []],
 
-        [['src/test/test-udev.c'],
+        [['src/test/udev-runner.c'],
```

Comment:
I do not like this change. I think all tests should be prefixed with 'test-'. As already commented, `test-udev` will be installed under `manual` directory, at least on fedora. So, it is safely excluded from the list in the driver provided by the later commit.